### PR TITLE
Don't throw runtime exception if WebSocket is already closed

### PR DESCRIPTION
**Pull request by satya164**
Refer #3364

### CODE REVIEW

**Comment by satya164** [comment]
cc @ide @mkonicek

File: Libraries/WebSocket/WebSocketBase.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ class WebSocketBase extends EventTarget {
       protocols = [];
     }
 
+    this.readyState === WebSocketBase.CONNECTING;
```
Comment (ide) [inline Libraries/WebSocket/WebSocketBase.js]: Assignment instead of comparison?

**Comment by ide** [comment]
Looks like a good improvement to me. @mkonicek?

**Comment by satya164** [comment]
@ide Yes, it's an assignment, the value should be "connecting" when it is trying to connect, then it'll change to "open", right now, it's not being done anywhere.

**Comment by ide** [comment]
Your commit uses === though is what I meant.

**Comment by satya164** [comment]
@ide Woah. So sorry. Fixed.

File: ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java [hunk 2]
```diff
@@ -132,7 +132,11 @@ public void close(int code, String reason, int id) {
     WebSocket client = mWebSocketConnections.get(id);
     if (client == null) {
       // This is a programmer error
```
Comment (mkonicek) [inline ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java]: Probably want to rewrite this comment, explaining why this is OK to ignore.

**Comment by mkonicek** [comment]
Besides the comment lgtm.

**Comment by satya164** [comment]
@mkonicek Fixed.

**Comment by mkonicek** [comment]
I still think the comment should explain why this would happen but just a nit so merging.

@facebook-github-bot shipit

**Pull request closed without merging**